### DISCUSSION TITLE
What is the best use of Prisma.ModelAggregateArgs?

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#15228] [Q&A]
In my `product.service.ts`:

```js
async getMany(args?: Prisma.ProductAggregateArgs): Promise<Product[]> {
return await this.prisma.product.findMany({
      ...args,
include: {
owner: true,
      },
    });
  }
```
but using ...args, throwing typescript error, complaining several incompatibilities.

Can anyone suggest how to use ModelAggregateArgs?

**Comment by nurul3101** [comment]
Hey @jUddinSazim 👋 
What is your use case? Do you want to get all the scalar fields of the product model here?
Can't you use `ModelScalarFieldEnum`?

**Reply by vladi-stevanovic** [reply]
👋 Hello @jUddinSazim have you perhaps had a chance to look into Nurul's question? Let us know if you still need help with this issue or if you've already solved it, we'd love to know how! It would be super useful for other users facing this problem!
